README.md

```diff
@@ -621,13 +621,38 @@ npm install prosemirror-utils
    ```
 
 
- * **`cloneRowAt`**`(cloneRowIndex: number) → fn(tr: Transaction) → Transaction`\
```

Comment:
you can't remove it, it'd be a breaking change

Reply:
I didn't remove, I moved to the bottom